k8s/neuvector/patches/prod/neuvector.yaml

```diff
@@ -6,7 +6,18 @@ metadata:
   namespace: neuvector
 spec:
   values:
-    some_value: true
+    rules:
```

Comment:
why not just add everywhere?

Reply:
Because I fear a storm of uncontrolled neuvector reloads ... let's do them one at a time!